rst_files/getting_started.rst

```diff
@@ -64,14 +64,21 @@ install `Anaconda <https://www.anaconda.com/download/#macos>`_ and then Julia
     * We do not recommend `JuliaPro <https://juliacomputing.com/products/juliapro.html>`_
       due to its limited number of available packages
 
-* Open Julia, by navigating to Julia through your menus or desktop icons
+* Open Julia by navigating to Julia through your menus or desktop icons or 
+  typing ``julia`` into a terminal or windows console
 
-Either way you should now be looking at something like this (modulo your operating system)
+Either way you should now be looking at something like this
 
 .. figure:: /_static/figures/julia_term_1.png
    :scale: 100%
 
-This is called the JULIA *REPL* (Read-Evaluate-Print-Loop), which we discuss more :ref:`below <julia_repl>`
+This is called the JULIA *REPL* (Read-Evaluate-Print-Loop)
+
+We examine the REPL and its different modes in more detail in the :doc:`tools and editors <tools_editors>` lecture  
+
+We'll start by installing some key Julia packages needed to run code in the lectures
+
+Typing ``]`` into the REPL switches to the package manager, where we can add packages
```

Comment:
@jlperla here is where the REPL is mentioned - I personally think this sufficient